tests/benchmarks/app_kernel/src/memmap_b.c

```diff
@@ -23,11 +23,15 @@ void memorymap_test(void)
 	u32_t et; /* elapsed time */
 	int i;
 	void *p;
+	int alloc_status;
 
 	PRINT_STRING(dashline, output_file);
 	et = BENCH_START();
 	for (i = 0; i < NR_OF_MAP_RUNS; i++) {
-		k_mem_slab_alloc(&MAP1, &p, K_FOREVER);
+		alloc_status = k_mem_slab_alloc(&MAP1, &p, K_FOREVER);
+		if (alloc_status != 0) {
+			break;
```

Comment:
shouldn't we fail the test instead?

Reply:
Agreed. I assumed the check_result() API would do that job.